apps/src/types/progressTypes.ts

```diff
@@ -59,6 +59,7 @@ export interface Level {
   title: number;
   url: string;
   status?: string;
+  usesLabContainer: boolean;
```

Comment:
Would making this `boolean | undefined` help with not having to update this in all the tests?

Reply:
Ah, it probably would.  That said, I thought it worth having the test reflect the fact that all levels now have this property.